tee-worker/client-api/parachain-api/prepare-build/interfaces/trustedCalls/definitions.ts

```diff
@@ -0,0 +1,103 @@
+export default {
```

Comment:
Maybe it would be worth to follow the convention and rename `trustedCalls` to `trusted-calls` ?

Reply:
It's good to keep it consistent with other file names. Renamed!

Reply:
We can only use underscore naming conventions [here](https://github.com/litentry/litentry-parachain/pull/2244/files#diff-180fa8477282b7aa025b54fe56b7abf976c06a87d0604d65e725c6a5e515d08fR3), as underscores are the only allowed characters. Please refer to the [documentation](https://polkadot.js.org/docs/api/examples/promise/typegen/#types-setup); consistency is a must.